packages/ui/src/components/va-chip/VaChip.vue

```diff
@@ -34,6 +34,10 @@
         v-if="closeable"
         class="va-chip__close-icon"
         name="close"
+        role="button"
+        aria-label="close"
+        aria-hidden="false"
+        :tabindex="indexComputed"
```

Comment:
Will be great if you name all tab indexes as you already named below (`tabIndexComputed`).